deutsche-gesellschaft-fur-psychologie.csl

```diff
@@ -279,6 +283,65 @@
       <text variable="locator" prefix=" "/>
     </group>
   </macro>
+  <macro name="issued-citation">
+    <choose>
+      <if variable="issued">
+        <group>
+          <choose>
+            <if variable="original-date">
+              <date variable="original-date">
+                <date-part name="year"/>
+              </date>
+              <choose>
+                <if variable="translator">
+                  <text value="; übers. "/>
+                </if>
+                <else>
+                  <text value="/"/>
+                </else>
+              </choose>
+            </if>
+          </choose>
+          <text macro="issued-year"/>
+        </group>
+      </if>
+      <else>
+        <group delimiter="-">
+          <text term="no date" form="short"/>
+          <text variable="year-suffix"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="original-date">
+    <date variable="original-date">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="publication-history">
+    <group prefix="(" suffix=")">
+      <choose>
+        <if variable="references">
+          <text variable="references"/>
+        </if>
+        <else>
+          <choose>
+            <if variable="original-date">
+              <group>
+                <text value="Original erschienen"/>
+                <text macro="original-date" prefix=" "/>
+                <choose>
+                  <if variable="translator">
+                    <text variable="original-title" prefix=": "/>
```

Comment:
I'm a bit confused by the prefix here. What's an example citation with a translator look like?

Reply:
From the manual (p. 123):
Lindsay, P. H. & Norman, D. A. (1981). _Einführung in die Psychologie_ (H.-D. Dumpert, F. Schmidt, M. Schuster & M. Steeger, Übers.). Berlin: Springer. (Original erschienen 1977: Human information processing).

Reply:
A classical work would not have the translator part neither an original title. Therefore the last bit would only state (Original erschienen 1930) for example.